runtime/lua/vim/lsp/handlers.lua

```diff
@@ -379,7 +379,7 @@ local make_call_hierarchy_handler = function(direction)
       for _, range in pairs(call_hierarchy_call.fromRanges) do
         table.insert(items, {
           filename = assert(vim.uri_to_fname(call_hierarchy_item.uri)),
-          text = call_hierarchy_item.name,
+          text = call_hierarchy_item.detail,
```

Comment:
```suggestion
          text = call_hierarchy_item.detail or call_hierarchy_item.name,
```

According to the specification, `detail` is optional

Reply:
Good catch, thanks!